gems/pending/util/vcenter_thumb_print.rb

```diff
@@ -0,0 +1,25 @@
+require 'util/thumb_print'
+
+class VcenterThumbPrint < ThumbPrint
```

Comment:
@chessbyte I didn't think it particularly fit in any other gems/pending subdirectory, and that it could conceivably be of general use in other places.  Your mileage may vary.  Do you have a suggestion as to another home for it?